package exception;

public class ExceptionExample {

	public static void main(String[] args) {
		
		System.out.println("main method start...");
		
		try
		{
			int i = 10/0; // arithmetic
			System.out.println(i);
		}
		catch (ArithmeticException e) {
			
		}
		
		
		System.out.println("main method end...");
		System.out.println("main method end...");
		System.out.println("main method end...");
		System.out.println("main method end...");
	}
}
